src/main/java/uk/gov/hmcts/reform/sscs/validators/FormTypeValidator.java

```diff
@@ -86,20 +86,19 @@ private synchronized Schema tryLoadSscsSchema(String schemaLocation) {
     }
 
     public String getFormType(String caseId, ExceptionRecord exceptionRecord) {
-        String formType = exceptionRecord.getFormType();
-        log.info("formtype for case {} is {}", caseId, formType);
+        ScannedData scannedData = sscsJsonExtractor.extractJson(exceptionRecord);
+        String formType = getField(scannedData.getOcrCaseData(), FORM_TYPE);
 
-        if (formType == null || notAValidFormType(formType)) {
-            ScannedData scannedData = sscsJsonExtractor.extractJson(exceptionRecord);
-            String ocrFormType = getField(scannedData.getOcrCaseData(), FORM_TYPE);
+        if (formType == null  || notAValidFormType(formType)) {
+            formType = exceptionRecord.getFormType();
 
-            if (ocrFormType == null || notAValidFormType(ocrFormType)) {
-                return null;
-            } else {
-                formType = ocrFormType;
+            if (formType == null || notAValidFormType(formType)) {
```

Comment:
Can you change to formType != null && notAValidFormType(formType) as there is no need to set the field to null if already null

Reply:
no, it is different, it checking the user input and the original form type , if both null, that means null, otherwise, if user has not input anything it will take the original form type.

Since there is the place have issue, I trying to keep the code structure and do less change to avoid another problem happen